pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -87,7 +88,11 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 	server.EventTTL = 2 * time.Hour
 	server.ServiceClusterIPRange = net.IPNet(flagtypes.DefaultIPNet(options.KubernetesMasterConfig.ServicesSubnet))
 	server.ServiceNodePortRange = *portRange
-	server.AdmissionControl = strings.Join(AdmissionPlugins, ",")
+	admissionPlugins := AdmissionPlugins
+	if len(options.KubernetesMasterConfig.AdmissionConfig.PluginOrderOverride) > 0 {
```

Comment:
I think our new override should take the place of anything set by the extended args, so this should move below the `cmdflag.Resolve`.